.gitlab/ci/targets.yml

```
@@ -15,30 +15,46 @@ deploy-sdk-apm821xx-sata:
 
 deploy-imagebuilder-ar71xx-generic:
   extends: .deploy-imagebuilder
+  except:
+    - master
```

From my understanding the `except` syntax is faded out in favor for `rules`. Should we directly switch to that instead?

```suggestion
  rules:
    - if: $CI_COMMIT_BRANCH != "master"
```

I did not yet test this change but it's maybe more future-proof.

---

https://docs.gitlab.com/ee/ci/yaml/#onlyexcept-basic

> Note: The rules syntax is an improved, more powerful solution for defining when jobs should run or not. Consider using rules instead of only/except to get the most out of your pipelines.